docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -254,6 +254,22 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
 
 .. _datetime mapping:
 
+.. _oracle-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in
+Oracle. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
+
+.. include:: sql-delete-limitation.fragment
+
```

Comment:
Move to the bottom of the article, replacing the old "these statements are not supported" section

Reply:
Okay.